src/Compilers/CSharp/Portable/BoundTree/Formatting.cs

```diff
@@ -120,6 +120,14 @@ public override object Display
         }
     }
 
+    internal partial class BoundDiscardedExpression
+    {
+        public override object Display
+        {
+            get { return string.Empty; }
```

Comment:
You were correct after all. Found a test where this Display output will be part of the message. Thanks

---
In reply to: [88765451](https://github.com/dotnet/roslyn/pull/15192#discussion_r88765451) [](ancestors = 88765451,88762755)